wazuh/wazuh-dashboard-plugins
# Fix maximized visualizations styles

### Description

Fixed the styles when maximising a visualisation as some were too big and others did not occupy the whole screen, the docked menu is taken into account.
 
### Issues Resolved
- #6859 

### Evidence

https://github.com/user-attachments/assets/ba4ef531-a2b7-464e-abb5-3c81e1bcd985

### Test

1. Navigate to the different dashboards and test maximising visualisations
2. Navigate to events and try maximising the displays
3. They should all be rendered accordingly and should take into account the docked menu or not.

### Check List
